framework/source/class/qx/event/message/Bus.js

```diff
@@ -8,8 +8,7 @@
      2007 Christian Boulanger
 
    License:
-     LGPL: http://www.gnu.org/licenses/lgpl.html
-     EPL: http://www.eclipse.org/org/documents/epl-v10.php
+     MIT: https://opensource.org/licenses/MIT
      See the LICENSE file in the project's top-level directory for details.
```

Comment:
There is no LICENSE file. There is a license.txt file which includes the former licenses. That should be one of the first files changed, and either it should be called LICENSE, or all of these individual file license changes should reference license.txt. (I think the .txt is important for viewing it on the browser from github, so probably the latter option -- correcting this line in each of the individual files -- is the better one.)